third_party/v8/src/base/logging.h

```diff
@@ -5,6 +5,7 @@
 #ifndef V8_BASE_LOGGING_H_
 #define V8_BASE_LOGGING_H_
 
+#include <cstdint>
```

Comment:
What's this for?

Reply:
Due to changes in inlcudes in base/ files, includes had to be added where they were actually needed